test/users.test.js

```diff
@@ -357,18 +353,82 @@ describe('users', () => {
           res.body.should.have.property('message');
           res.body.should.have.property('internal_code');
 
-          res.body.message.should.be.equal('Error verifying hash'); // TODO
+          res.body.message.should.be.equal('jwt malformed');
+        })
+        .then(() => done());
+    });
+    it('should fail using an expired token', done => {
+      const authorization = usersHelper.expiredUser();
+      chai
+        .request(server)
+        .post('/admin/users')
+        .set(sessionManager.HEADER_NAME, authorization)
+        .send({
```

Comment:
you repeat this request body.
why do you don't put this request body in a helper?